web_monitoring/filtering.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+from urllib.parse import urlparse
+from bs4 import BeautifulSoup as BSoup
+
+day_list = ['mon','tue','wed','thu','fri','sat','sun']
+month_list = ['jan','feb','mar','apr','may','jun','jul','aug','sep','oct','nov','dec']
+tag_list = ['<td class="c" id="displayMonthEl"','<td class="c" id="displayDayEl"','<td class="c" id="displayYearEl"']
+social_media = ['twitter.com','platform.twitter.com']
+
+
+def df_filter(df):
+
+    df['review'] = 'yes'
+    df['id'] = 'none'
+    df['priority'] = 1.0
+    df['annotation'] = 'no annotation'
+    
+    for index,row in df.iterrows():
+        if((str(row['new']).lower() in month_list) and (str(row['old']).lower() in month_list)):
+            df.loc[index]=df.loc[index].replace(df.loc[index]['review'],'no')
```

Comment:
@mhucka As you may have noticed, I have followed these rules for some other parts of the code. I'll make sure I do a style and readability check before pushing code.